doc/3/controllers/security/create-user/index.md

```diff
@@ -0,0 +1,54 @@
+---
+code: true
+type: page
+title: createUser
+description: Security:createUser
+---
+
+# createUser
+
+Creates a new user in Kuzzle.
+
+:::info
+There is a small delay between user creation and its availability in our search layer (usually a couple of seconds).
+That means that a user that was just created may not be returned by the `searchUsers` function at first.
+:::
+
+---
+
+## createUser(id, user, [options], [callback])
+
+| Arguments  | Type        | Description                                                                                                                  |
+| ---------- | ----------- | ---------------------------------------------------------------------------------------------------------------------------- |
+| `id`       | string      | [Unique user identifier](/core/1/guides/essentials/user-authentication#kuzzle-user-identifier-kuid)                   |
```

Comment:
Argument types must be embedded in `<pre>` tags.

Reply:
nope

Reply:
Why? All our "recent" documentation pages are formatted like this.

Reply:
sorry dude, I was in a hurry. Actually, Vuepress doesn't really like the HTML tags into the markdown, so to format the types, we should use the backquotes, like in normal markdown syntax.

Reply:
according to @xbill82 , I should use backquotes instead

Reply:
@xbill82 I don't get it: all our recent documentation pages are written following that model. And backticks and pre tags doesn't output the same thing: the former outputs a code-formatted snippet, while the latter only applies a fixed-width font to the tagged string, making things as `string[]` more readable (vs. "string[]")

If tags are a problem, is there a way to add some kind of keyword allowing us to print something in fixed-width?

Reply:
@scottinet is right, we always put arguments types between `<pre>` tag
https://raw.githubusercontent.com/kuzzleio/sdk-javascript/master/doc/7/controllers/document/create/index.md
Which look like this:https://docs.kuzzle.io/sdk/js/7/controllers/document/create/

Reply:
It has been corrected :+1: